slimevr-setup.md

```diff
@@ -22,10 +22,20 @@ This guide should help you set up SlimeVR trackers and software.
 * Download the latest [SlimeVR Server zip](https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR.zip) and unpack it.
 * Turn each tracker on and see if they work.
 
-Each tracker should blink a LED briefly on startup, and then blink every 10-15 seconds once when the tracker is on. Any other lights (for example, always on, blinking multiple times, etc.) mean something is wrong.
+Each tracker should blink a LED briefly on startup, and then blink every 10 seconds to indicate its status as follows:
```

Comment:
```suggestion
Each tracker should blink a LED briefly on startup, and then blink every few seconds to indicate its status as follows:
```

It blinks every 10 seconds only in standbuy, errors signals have shorter wait times.

Reply:
Done